src/automation/azext_automation/manual/_help.py

```diff
@@ -0,0 +1,221 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+helps['automation account'] = """
```

Comment:
just curious, why this isn't generated but manual?

Reply:
I hide `account` and `job` in the generated code. The reason is that we need to update `account create` command (remove some parameters). Currently we can't hide `create_or_update` operations via codegen's .md file, thus the default `account create` will always have some parameters which should be hidden. So currently the only solution is to hide all the command group and manually implement it.